OpenGov-OpenData/ckanext-custom-schema
# [PA-647] Add data dictionary to cademo schema

## [TICKET](https://opengovinc.atlassian.net/browse/PA-647)

## Description

This PR adds the data dictionary table to cademo schema.

## Testing
- Install and enable [ckanext-scheming](https://github.com/OpenGov/ckanext-scheming)
- Install ckanext-custom-schema and use cademo branch. The plugin name will be `custom_schema`
- Check that the data dictionary table does NOT appear on the resource read page of a tabular resource
- Checkout this PR and restart CKAN
- Check that the data dictionary table now appears on the resource read page of a tabular resource
